tensorboard/plugins/profile/README.md

```diff
@@ -117,3 +118,14 @@ are defined below.
 *   selfTimeInMs: number
 *   selfTimeInPercent: number
 *   category: string
+
+#### 2.3 Recommendation DataTable
+
+2.3.1 Properties:
+
+*   overall
```

Comment:
nit: At some point (not necessarily in this PR), it might behove us to document these properties. ie, what does the `overall` property contain? What is its type?

Reply:
Done.